behat_ibexa_commerce.yaml

```diff
@@ -1,6 +1,7 @@
 imports:
     - behat_ibexa_experience.yaml
     - vendor/ibexa/payment/behat_suites.yml
+    -  vendor/ibexa/storefront/behat_suites.yml
```

Comment:
One addtional space to remove here.

Reply:
Fixed in: https://github.com/ibexa/behat/pull/79/commits/044af2e92956a9422fd15c4645a1eaa31d41aa94